pkg/webhook/webhook.go

```diff
@@ -494,6 +516,15 @@ func bindWebhookConfigToFleetSystem(ctx context.Context, k8Client client.Client,
 	return nil
 }
 
+func getGeneralRulesForNamespacedResources(cudOperations []admv1.OperationType, scope admv1.ScopeType) []admv1.RuleWithOperations {
```

Comment:
Hi Arvind! A nit here: with generic matchers, is this method still needed? Also, since the method is already for namespaced resources, there's no need to pass in scope I assume?